src/main/app/src/static/smart-documents-result.html

```diff
@@ -283,6 +293,17 @@
         document.getElementById("sub-text").textContent = subText;
       }
 
+      function getTypeValue(zaakValue, taakValue, language) {
+        const typeMap = {
+          en: { zaak: "case", taak: "task" },
```

Comment:
Perhaps we should keep this `zaak` in english too

Reply:
You mean we should use the EN equivalent in the query parameters? 'case' ?

Reply:
query should be zaak. That's a contract between frontend and backend.

But if this is used for text on the page we can still use zaak I think.